deployments/tike/config/common.yaml

```diff
@@ -52,7 +52,9 @@ jupyterhub:
             - "-c"
             - >
               rsync -r /etc/default-home-contents/ /home/jovyan;
-              gitpuller https://github.com/spacetelescope/tike_content master /home/jovyan/tike_content
+              gitpuller https://github.com/spacetelescope/tike_content master /home/jovyan/tike_content;
```

Comment:
We should start a setup-notebooks for tike so that this stuff can be installed in the standalone image as well.   setup-notebooks may really be too narrow a name,   it's more like setup-home.    And I think maybe  ALL of that goes into setup-home,  including the rsync,  because for the standalone image it only gets called when the image is built.   I'm fine with merging this but think we should continue evolving setup-notebooks/setup-home.

Reply:
You're right, that's a good idea.  I might do that now.  I like `setup-home`.  Not having things in the Docker image as well also impact security scanning.

Reply:
Nice point about security!